bindings/python/py_src/safetensors/flax.py

```diff
@@ -98,7 +98,7 @@ def load(data: bytes) -> Dict[str, jnp.DeviceArray]:
     return _np2jnp(flat)
 
 
-def load_file(filename: Union[str, os.PathLike]) -> Dict[str, jnp.DeviceArray]:
+def load_file(filename: Union[str, os.PathLike], device: str = None) -> Dict[str, jnp.DeviceArray]:
```

Comment:
If putting a type annotation, this is an `Optional[str]` (here and everywhere below).